CHANGELOG.md

```diff
@@ -5,12 +5,6 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [1.0.0] - 2020-02-20
+## [1.0.0] - 2020-10-29
 
 ### Added
```

Comment:
Do we not need to mention what all has been added in this version? Why did we remove the items present earlier?

Reply:
This "Added" section only begins after the solution is published

Reply:
Got it.